examples/command_error.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.message_content = True
+
+positive_emoji: str = "\U00002705"
```

Comment:
Maybe switch these constants to SCREAMING_SNAKE_CASE?
Also the typehint is unnecessary.